### DISCUSSION TITLE
about mhm installation in ubuntu

### DISCUSSION THREAD

**Discussion by manisasah** [mhm-ufz/mHM#15] [Installation] [answered]
hello,
while installing mhm using ubuntu.i got stuck in ./bootstrap process.how can i solve the following error??
Error when bootstrapping CMake:
```bash
Cannot find a C++ compiler that supports both C++11 and the specified C++ flags.
Please specify one using environment variable CXX.
The C++ flags are "".
They can be changed using the environment variable CXXFLAGS.
See cmake_bootstrap.log for compilers attempted.
---------------------------------------------
Log of errors: /home/decode/cmake-3.16.3/Bootstrap.cmk/cmake_bootstrap.log
```

**Comment by MuellerSeb** [comment]
Hey @manisasah 

This will be the exact topic of today's User-Forum at 13.30 CEST :relaxed:
We will discuss this and put some guides online afterwards.

Best,
Sebastian

**Reply by pallavhydro** [reply]
Hi Manisha

If you are still struggling with mHM installation in your Ubuntu OS, I'd suggest you follow instructions from this page -

https://github.com/mhm-ufz/mhm/wiki/Installing-on-Windows-10-using-WSL2-and-Conda

Since you already are using Ubuntu, you can directly jump to the section **Setting mHM environment with Conda in the new Linux**. 

Kindly update us about how it goes.

Cheers
Pallav

**Comment by manisasah** [comment]
Thanks
Will try that way.


With regards,
Manisha sah

**Comment by MuellerSeb** [comment, marked as answer]
You can find all installation guides here: https://mhm-ufz.org/guides/
